packages/std-contracts/src/test/MudTest.t.sol

```diff
@@ -6,6 +6,7 @@ import { IWorld } from "solecs/interfaces/IWorld.sol";
 import { IUint256Component } from "solecs/interfaces/IUint256Component.sol";
 import { Vm } from "forge-std/Vm.sol";
 import { getAddressById } from "solecs/utils.sol";
+import { SystemStorage as s } from "solecs/SystemStorage.sol";
```

Comment:
Any reason we can't keep this as `SystemStorage` instead of aliasing to `s`? Feels more readable.

Reply:
Handled here: [4859a07](https://github.com/latticexyz/mud/pull/264/commits/4859a0757c204c1b60cea8b97e1cfc4b66d7bc86) !